yarn.lock

```diff
@@ -14,14 +14,14 @@
   dependencies:
     "@types/angular" "*"
 
-"@types/angular@*", "@types/angular@1.6.26", "@types/angular@^1.6.15", "@types/angular@^1.6.16":
-  version "1.6.26"
-  resolved "https://registry.yarnpkg.com/@types/angular/-/angular-1.6.26.tgz#a96cccef648de0bf7435f3b9bad13f2430719305"
-
-"@types/angular@^1.6.28":
```

Comment:
curious how the `yarn.lock` file got changed without any corresponding `package.json` file changes (just curious, not asking for changes or investigation)

Reply:
Those are the same changes that broke the deb build the other day (https://github.com/spinnaker/deck/pull/4136). So they probably shouldn't go back in.